docs/source/getting_started/basic_functionality.rst

```diff
@@ -129,3 +108,51 @@ In order for the delete to be successful, the dataset must be empty.
 .. code-block:: python
 
     project.datasets.delete(id)
+
+Data Model Object Specific Methods
+-----------------------------------
+
+The client supports additional methods on certain data model object resources.
+
+Project-Wide Collections
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Data model objects are each contained in exactly one dataset.
+However, most methods that read data model objects can be evaluated against all of the datasets that are readable from a project.
+Special data model object collections are available in the project to support this, for example:
+
+.. code-block:: python
+
+    project.material_specs.list()
+
+Any method that writes to a data model object must be taken from within a dataset-scoped collection.
+
+Filter
+^^^^^^
+
+Filter lies somewhere between Get and List, returning only those objects that meet a given criterion.
+There are currently three ways to filter, and they work on each data model object type:
+:func:`~citrine.resources.data_concepts.DataConceptsCollection.filter_by_tags`,
+:func:`~citrine.resources.data_concepts.DataConceptsCollection.filter_by_attribute_bounds`,
+and :func:`~citrine.resources.data_concepts.DataConceptsCollection.filter_by_name`.
+
+Filtering by tags or attribute bounds can be scoped to a dataset or to a project.
+Filtering by name must be scoped to a dataset.
+
+Get Material History
+^^^^^^^^^^^^^^^^^^^^
+
+Starting with a specific root :class:`MaterialRun <citrine.resources.material_run.MaterialRun>`,
+you can retrieve the complete material history--every process, ingredient and material that went
+into the root material, as well as the measurements that were performed on all of those materials
+The method is :func:`~citrine.resources.material_run.MaterialRunCollection.get_history`,
+and it requires you to know a unique identifier (scope/id pair) for the material.
+
+LinkByUID
+^^^^^^^^^^
```

Comment:
I don't think this belongs under "Data Model Object Specific Methods"

Reply:
Do you have a suggestion as to where to put it?

Reply:
`getting_started/data_model`? It's not really a pice of functionality. I suppose it's not strictly part of the data model either, but is a fairly integral piece of its functioning